src/rewriter.coffee

```diff
@@ -118,14 +118,21 @@ exports.Rewriter = class Rewriter
   # The lexer has tagged the opening bracket of an indexing operation call.
   # Match it with its paired close.
   closeOpenIndexes: ->
+    startToken = null
```

Comment:
Allowing static computed class properties/methods was a bit more complex. In addition to the new grammar rule above, in the rewriter here we first want to make sure that the `[`/`]` in eg `@[b]: ...` are not tagged as `INDEX_START`/`INDEX_END` since that would be inaccurate and creates an ambiguous grammar

I believe this is safe because there's not a scenario where an actual `INDEX_END` would be immediately followed by a `:` token